drivers/auxdisplay/auxdisplay_jhd1313.c

```diff
@@ -298,14 +299,15 @@ static int auxdisplay_jhd1313_initialize(const struct device *dev)
 	auxdisplay_jhd1313_reg_set(config->bus.bus, 0x08, 0xAA);
 
 	/* Now set the background colour to white */
-	LOG_DBG("Background set to off");
-	auxdisplay_jhd1313_backlight_set(dev, 0);
+	LOG_DBG("Background set to white");
+	auxdisplay_jhd1313_backlight_set(dev, 1);
```

Comment:
@gmarull what are your thoughts on this? I don't know what this display looks like or if the text is readable with the background off, but I don't see why it should default to white (unless it's not readable) otherwise someone wanting a different default colour will have a flash of white until their application boots and they can set it to their desired colour

Reply:
Happy to set to black (it is pretty readable even with no backlight). The main reason I suggested the fix was to align with the code comment on the line before that mentioned white :)

Reply:
addressed in latest push (only updated the comment that improperly indicated "white")